src/_incydr_cli/cursor.py

```diff
@@ -88,7 +88,7 @@ def get_items(self, cursor_name):
 
     def replace_items(self, cursor_name, new_events):
         """
-        Used with alerts and audit_log events to avoid duplicates
+        Used with alerts audit_log events, and sessions to avoid duplicates
```

Comment:
Minor if you're back in here:

```suggestion
        Used with alerts, audit_log events, and sessions to avoid duplicates
```